src/App.tsx

```diff
@@ -1,61 +1,34 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { NewTodo } from './components/NewTodo/NewTodo';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  function addNewTodo(title: string, userId: number, completed = false) {
+    const maxId = Math.max(...todos.map(item => item.id));
+
+    const newT: Todo = {
+      id: maxId + 1,
+      title: title,
+      completed: completed,
+      userId: userId,
+    };
```

Comment:
[CODE STYLE] - The variable 'newT' is not descriptive. Consider renaming it to something more meaningful, such as 'newTodo' to clearly represent its purpose.